appveyor.bat

```diff
@@ -318,23 +326,27 @@ copy /Y %DEPENDENCIES%\gettext64\libintl-8.dll       ..\runtime\GvimExt64\
 copy /Y GvimExt32\*.*                    ..\runtime\GvimExt32\
 copy /Y %DEPENDENCIES%\gettext32\libiconv-2.dll      ..\runtime\GvimExt32\
 copy /Y %DEPENDENCIES%\gettext32\libintl-8.dll       ..\runtime\GvimExt32\
-rem copy /Y c:\gettext32\libgcc_s_sjlj-1.dll ..\runtime\GvimExt32\
-rem libgcc_s_sjlj-1.dll is not needed anymore. Don't include it in the installer.
-if exist %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll del %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll
+@rem copy /Y c:\gettext32\libgcc_s_sjlj-1.dll ..\runtime\GvimExt32\
+@rem libgcc_s_sjlj-1.dll is not needed anymore. Don't include it in the installer.
+@rem if exist %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll del %DEPENDENCIES%\gettext32\libgcc_s_sjlj-1.dll
 copy /Y ..\..\diff.exe ..\runtime\
 copy /Y %DEPENDENCIES%\gettext%BIT%\libiconv-2.dll   ..\runtime\
 copy /Y %DEPENDENCIES%\gettext%BIT%\libintl-8.dll    ..\runtime\
-rem if exist c:\gettext%BIT%\libgcc_s_sjlj-1.dll copy /Y c:\gettext%BIT%\libgcc_s_sjlj-1.dll ..\runtime\
+@rem if exist c:\gettext%BIT%\libgcc_s_sjlj-1.dll copy /Y c:\gettext%BIT%\libgcc_s_sjlj-1.dll ..\runtime\
 copy /Y winpty* ..\runtime\
 copy /Y winpty* ..\..\
-set dir=vim%TAG_NAME:~1,1%%TAG_NAME:~3,1%
+set "dir=vim%TAG_NAME:~1,1%%TAG_NAME:~3,1%"
 mkdir ..\vim\%dir%
-xcopy ..\runtime ..\vim\%dir% /Y /E /V /I /H /R /Q
+xcopy ..\runtime ..\vim\%dir% /Y /E /V /I /H /R /Q /EXCLUDE:..\..\exclist.txt
 :: need to escape the ! because of delayedexpansion being active
-7z a ..\..\gvim_%TAG_NAME:~1%_%ARCH%.zip ..\vim -x^^!vim\%dir%\syntax\testdir\
```

Comment:
why did you remove the syntax\testdir? ah, its in the exclude list

Reply:
Yes.